CONTRIBUTING.MD

```diff
@@ -0,0 +1,60 @@
+# Contribution Guidelines
+
+First off, thank you for thinking of contributing to this project! 🥳
+
+The following is a set of guidelines for contributing to Ona Data. Following these guidelines helps to communicate that you respect the time of the maintainers and your intent to help develop this project.
+
+## How to contribute
+
+### 1. Report a bug 🐛
```

Comment:
love the emoticon, I've always liked calling out bugs as errors, e.g. `Report an error/bug` but I get this is a term of art so this is cool as is